### PR TITLE
Sprite display

### PR DESCRIPTION
**Pull request by MattEqualsCoder**
Some notes with this:

- On installs, the sprite folder will be deleted and recreated. I had to do this because a lot of the sprites I automated had slightly different file names, so it was creating dupes.
- To counter this, folks can create a Sprites folder at %localappdata%/SMZ3Randomizer/Sprites/<SpriteType> to house sprites.
- With some of the Sprite class changes, there was an issue where it would say it had the previous sprite selected before this version but it wouldn't actually apply it. I changed the property names to force people to select a new sprite for the first time
- Right now you do have to select random each time. I was worried how it would work with multiworlds due to it sending the configs ahead of time and generating after the fact. I'll try to investigate more into not having to select random each time in a future update.
- Kinda snuck in here because skennedy brought up an issue with the plando. It was having an issue where it wasn't working due to not being able to create hint tiles. For now, I'm removing the hint tiles for plandos and opened an issue to allow people to write up hint tiles for plandos

### CODE REVIEW

File: src/Randomizer.App/BoolToVisibilityConverter.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+public class BoolToVisibilityConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return (bool) value ? Visibility.Visible : Visibility.Collapsed;
+    }
+
+    public object ConvertBack(object value, Type targetType, object parameter,CultureInfo culture)
+    {
+        throw new NotImplementedException();
```
Comment (CPColin) [inline src/Randomizer.App/BoolToVisibilityConverter.cs:18]: Guessing we never plan to convert back to a `bool`?

**Review by CPColin — APPROVED** [review]

File: src/Randomizer.App/BoolToVisibilityConverter.cs (hunk 1 above)
Comment (MattEqualsCoder) [reply src/Randomizer.App/BoolToVisibilityConverter.cs:18]: I don't think the other way would make sense, really. I'm not sure what two way binding you could do would change between the enum visibility values and flow well from a user perspective.

I just use this as an easy way to hide stuff based on booleans.

Comment (CPColin) [reply src/Randomizer.App/BoolToVisibilityConverter.cs:18]: Gotcha, makes sense to me!

File: src/Randomizer.App/SpriteService.cs [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    /// Loads a ship ips sprite
+    /// </summary>
+    /// <param name="path">The path to the ips file</param>
+    /// <returns>The sprite object</returns>
+    private Sprite LoadIpsSprite(string path)
+    {
+        var file = new FileInfo(path);
+        var parts = Path.GetFileNameWithoutExtension(path).Split(" by ");
+        var name = parts[0];
+        var author = parts[1];
```
Comment (Vivelin) [inline src/Randomizer.App/SpriteService.cs:128]: This'll throw if someone drops in an incorrectly named file. Did you handle that elsewhere?

**Review by Vivelin — DISMISSED** [review]

**Review by CPColin — APPROVED** [review]

**Pull request merged**